config/lib/seed.js

```diff
@@ -89,12 +95,16 @@ function seedTheUser (user) {
 
 // report the error
 function reportError (err) {
-  console.log();
-  console.log(err);
-  console.log();
+  if (seedOptions.logResults) {
+    console.log();
+    console.log(err);
+    console.log();
+  }
 }
 
-module.exports.start = function start() {
+module.exports.start = function start(options) {
+  if (options && options.hasOwnProperty('logResults')) seedOptions.logResults = options.logResults;
```

Comment:
How about using lodash's .has method? it's a shorthand way of doing it and we can use it consistently across the project?